microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/core/jaxrs/JaxRsContext.java

```diff
@@ -49,4 +68,18 @@ public static JaxRsContext getJaxRsContext(JavaCodeLensContext context) {
 		}
 		return jaxRsContext;
 	}
+	
+	/** Create the local base URL 
+	 * 
+	 * @return the String representation of the project base URL
+	 */
+	public String getLocalBaseURL() {
+	  StringBuilder localBaseURL = new StringBuilder("http://localhost:");
+	  localBaseURL.append(getServerPort()).toString();
+	   if (rootPath != null) {
+	      localBaseURL.append(getRootPath());

```

Comment:
What is the behavior of quarkus if root path doesn't starts with '/' ?

Reply:
It will throw an error on build:
![Screenshot from 2021-09-07 13-23-30](https://user-images.githubusercontent.com/26389510/132386007-af7ee3dd-c01e-4f05-9c9d-276a74c17a66.png)

Reply:
Ok thanks for the feedback. Could you create an issue to validate this property on application properties file please

Reply:
Are we good to close this one then?

Reply:
Yes let's keep like this.

Reply:
If we're good to close, can we merge this and I'll open another pr to handle the 2 cases:
- there is no option to work with host differing from localhost. That's a problem.
- there is no option to work with https:// at all. Bad, but can be fixed via SSL termination or/and proxy.